Sources/StreamChat/Models/Poll.swift

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The model for a Poll.
+public struct Poll: Equatable {
+    /// A boolean indicating whether the poll allows answers.
+    public let allowAnswers: Bool
+    
+    /// A boolean indicating whether the poll allows user-suggested options.
+    public let allowUserSuggestedOptions: Bool
+    
+    /// The count of answers received for the poll.
+    public let answersCount: Int
+    
+    /// The date and time when the poll was created.
+    public let createdAt: Date
+    
+    /// A brief description of the poll.
+    /// This property is optional and may be `nil`.
+    public let pollDescription: String?
+    
+    /// A boolean indicating whether the poll enforces unique votes.
+    public let enforceUniqueVote: Bool
+    
+    /// The unique identifier of the poll.
+    public let id: String
+    
+    /// The name of the poll.
+    public let name: String
+    
+    /// The date and time when the poll was last updated.
+    /// This property is optional and may be `nil`.
+    public let updatedAt: Date?
+    
+    /// The count of votes received for the poll.
+    public let voteCount: Int
+    
+    /// A dictionary containing custom fields associated with the poll.
+    public let custom: [String: RawJSON]
+    
+    /// A dictionary mapping option IDs to the count of votes each option has received.
+    /// This property is optional and may be `nil`.
+    public let voteCountsByOption: [String: Int]?
+    
+    /// A boolean indicating whether the poll is closed.
+    public let isClosed: Bool
+    
+    /// The maximum number of votes allowed per user.
+    /// This property is optional and may be `nil`.
+    public let maxVotesAllowed: Int?
+    
+    /// A string representing the visibility of the voting process.
+    /// This property is optional and may be `nil`.
+    public let votingVisibility: String?
```

Comment:
Don't we have a data structure to represent the `VotingVisiblity`? 🤔

Reply:
yep, forgot to change it, thx